tests/modules/Instabug.spec.js

```diff
@@ -149,6 +150,7 @@ describe('Instabug Module', () => {
     expect(NativeInstabug.init).toBeCalledWith(
       instabugConfig.token,
       instabugConfig.invocationEvents,
+      instabugConfig.codePushLabel,
```

Comment:
I think we should also test the case when the `codePushLabel` is not passed and verify that it was called with an empty string `''`.

Reply:
Makes sense! I added another test for `init` to verify the native method is called with the default values given the optional properties aren't passed.